Golden_Path.postman_collection.json

```diff
@@ -204,6 +204,7 @@
 							}
 						}
 					],
+					"protocolProfileBehavior": {},
```

Comment:
what does this mean? lets discuss

Reply:
This is automatically added by Postman and no change to functionality as properties are empty.